### DISCUSSION TITLE
[Question] Getting this error in 0.5. Please assist

### DISCUSSION THREAD

**Discussion by gridsin** [kubilus1/autoortho#301] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [e.g. Windows 11]
- XPlane Version: [e.g. XP12.06 beta 2]
- AutoOrtho Version [e.g. 0.5]

### question

I am attaching a screenshot and log of the error. This is a clean install of autoortho and Xplane. I deleted all previous scenery packages.
[autoortho.log](https://github.com/kubilus1/autoortho/files/12229275/autoortho.log)
<img width="668" alt="Screenshot 2023-08-01 172120" src="https://github.com/kubilus1/autoortho/assets/141147402/491d06b0-f26f-49f6-8f0c-a6bc07ecd3ef">

**Comment by chezhj** [comment]
Not sure but it seems that ao can't properly load the FUSE libraries. You installed winfsp? Try reinstalling, reboot.
currently I have both Dokan and winFSP installed. Because with only one, I rand into issues.
